libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/ExitSignCreator.java

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.services.android.navigation.ui.v5.instruction;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.mapbox.api.directions.v5.models.BannerComponents;
+import com.mapbox.services.android.navigation.ui.v5.R;
+
+import java.util.List;
+
+public class ExitSignCreator extends NodeCreator<BannerComponentNode, ExitSignVerifier> {
+  private String exitNumber;
+  private int startIndex;
+  private TextViewUtils textViewUtils;
+  private String modifier;
+
+  ExitSignCreator() {
+    super(new ExitSignVerifier());
+    textViewUtils = new TextViewUtils();
+
+  }
+
+  @Override
+  BannerComponentNode setupNode(BannerComponents components, int index, int startIndex, String
+    modifier) {
+    if (components.type().equals("exit")) {
+      return null;
```

Comment:
Why do we return `null` here?  Not the same data as `"exit-number"`?

Reply:
@danesfeder this contains the text in the native language representing "exit", but for parity with iOS we are dropping this node (See screenshots, it doesn't say `exit`, it just says the exit number)